src/services/event.ts

```diff
@@ -191,7 +200,16 @@ async function findEventHelperBlob({
       {
         model: SessionReportPilot,
         as: 'sessionReports',
-        order: [['data.startDate', 'ASC'], ['data.title', 'ASC']],
+        separate: true, // This is required to order the joined table results.
+        attributes: [
+          'id',
+          'eventId',
+          'data',
+          'createdAt',
+          'updatedAt',
+          [sequelize.literal('Date(CASE WHEN "SessionReportPilot".data->>\'startDate\' = \'\' THEN NULL ELSE "SessionReportPilot".data->>\'startDate\' END)'), 'startDate'],
```

Comment:
Please use backticks and add newlines so it is more readable and then the single quotes don't need to be escaped

Reply:
Also, don't use CASE for this, use NULLIF(<value>, ''), as this is far easier to read and is much faster